modules/virt-creating-storage-class-legacy-hpp.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-configuring-local-storage-for-vms.adoc
+
+:_content-type: PROCEDURE
+[id="virt-creating-storage-class-legacy-hpp_{context}"]
+= Creating a storage class for the legacy hostpath provisioner
+
+You create a storage class for the legacy hostpath provisioner (HPP) by creating a `StorageClass` object without the `storagePool` parameter.
+
+.Procedure
+
+. Create a `storageclass.yaml` file to define the storage class:
++
+[source,yaml]
+----
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: hostpath-provisioner <1>
+provisioner: kubevirt.io/hostpath-provisioner
+reclaimPolicy: Delete <2>
+volumeBindingMode: WaitForFirstConsumer <3>
+----
+<1> Assign any meaningful name to the storage class. In this example, `csi` is used to specify that the class is using the CSI provisioner, instead of the legacy provisioner. Choosing descriptive names for storage classes, based on legacy or CSI driver provisioning, eases implementation of your migration strategy.
```

Comment:
> In this example, `csi` is used

This is the legacy YAML example, so the callout should be updated

Reply:
Good catch. That must have been in the original and I just copied it over.